### PR TITLE
Work on userguide and get the docs ready to be consumed by the docker docs-release

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@docker.com>

Add docker.docker.com metadata and reflow to 80-chars for GH diffs

for https://github.com/docker/docker/pull/10173

see http://dowideit-docs.s3-website-ap-southeast-2.amazonaws.com/ for a preview.

### CODE REVIEW

**Comment by vieux** [comment]
@SvenDowideit thanks, should we move all our doc in one document ? We we will have only one article in docs.docker.com ?

**Comment by vieux** [comment]
@SvenDowideit all the relative links in http://dowideit-docs.s3-website-ap-southeast-2.amazonaws.com/swarm/README/ are broken.

**Comment by SvenDowideit** [comment]
@vieux :( For this commit, I'm trying to not break anything in your repo - I think I need to add some script to convert those on my end afterwards.

**Comment by vieux** [comment]
@SvenDowideit it's unpractical to use our READMEs for now. 

We are going to create a doc folder avec one or two .md inside and you'll be able to use that,

OK for you ?

**Comment by vieux** [comment]
Ok, we can't merge the PR as is because it adds `mkdoc` medata in all our README.md.

Also we have the travis badge and our names in the UserGuide.

What about we merge https://github.com/docker/swarm/tree/doc/doc and you use those files instead ?

**Comment by SvenDowideit** [comment]
@vieux is this better?

**Comment by SvenDowideit** [comment]
@vieux PTAL - I'd love this to start hitting docs.master.dockerproject.com

**Comment by aluzzardi** [comment]
@SvenDowideit The commits are not signed :(

File: README.md [hunk 1]
```diff
@@ -49,33 +57,36 @@ $ docker -H tcp://<swarm_ip:swarm_port> logs ...
...
+Please refer to the [Docker documentation](https://docs.docker.com/articles/https/)
+for more information on how to set up TLS authentication on Docker and generating
+the certificates.
 
 Note that Swarm certificates must be generated with`extendedKeyUsage = clientAuth,serverAuth`.
 
 ## Participating
 
-We welcome pull requests and patches; come say hi on IRC, #docker-swarm on freenode.
+We welcome pull requests and patches; come say hi on IRC, #swarm on freenode.
```
Comment (vieux) [inline README.md]: it's #docker-swarm here :D

File: userguide.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+> port that the Swarm manager can access.
+
+Docker `swarm` is currently only available as a single go binary on Linux. Download
+it from [the latest release](https://github.com/docker/swarm/releases/latest) page
+on GitHub.
+
+For example:
+
+```
+	$ wget -O swarm https://github.com/docker/swarm/releases/download/v0.1.0-rc1/swarm-Linux-x86_64
```
Comment (vieux) [inline userguide.md]: docker-swarm here

File: userguide.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+```bash
+# create a cluster
+$ swarm create
+6856663cdefdec325839a4b7e1de38e8 # <- this is your unique <cluster_id>
+
+# For each of your nodes, start a swarm agent
+#  the Docker daemon <node_ip> doesn't have to be public (eg. 192.168.0.X),
+#  as long as the swarm manager can access it.
+$ swarm join --addr=<node_ip:2375> --discovery token://<cluster_id>
```
Comment (vieux) [inline userguide.md]: we removed the `--discovery` flag

Comment (SvenDowideit) [inline userguide.md]: its in the preview release, which is what i'm documenting atm.

will we be getting an rc2 this week?

Comment (vieux) [inline userguide.md]: yes, tomorrow probably

Comment (SvenDowideit) [inline userguide.md]: ok, I'll leave this PR til then and re-visit the docs for rc2 :)

**Comment by vieux** [comment]
@SvenDowideit rc2 was released :D

**Comment by SvenDowideit** [comment]
yup, I'll be updating today, ready for you guys tomorrow :)

**Comment by SvenDowideit** [comment]
@vieux @aluzzardi, the Boot2docker section in the userguide.md, I will add something like:


```
diff --git a/userguide.md b/userguide.md
index 529ab94..2f46ebd 100644
--- a/userguide.md
+++ b/userguide.md
@@ -102,6 +102,60 @@ the certificates.
 
 ## Boot2Docker quickstart
 
+Download and install the OS X Swarm binary, and create a Swarm cluster id:
+
+```
+$ curl -Ssl https://github.com/docker/swarm/releases/download/v0.1.0-rc2/docker-swarm_darwin-amd64 > swarm
+$ chmod 755 swarm
+$ sudo mv swarm /usr/local/bin/swarm
+$ swarm create > ~/.docker/swarm-token
+```
+
+Make sure your Boot2Docker virtual machine is up and running
+
+```
+$ boot2docker init
+$ boot2docker up
+...
+$ $(boot2docker shellinit)
+```
+
+Start the Docker Swarm node agent:
+
+```
+$ docker run -d -it --net host --name swarm-node svendowideit/docker-swarm \
+         /swarm join --addr=$(boot2docker ip):2376 token://$(cat ~/.docker/swarm-token)
+```
+
+and a Docker Swarm manager daemon:
+
+```
+$ docker run -d -it --net host -v /var/lib/boot2docker/tls:/tls\
+       --name swarm-manager svendowideit/docker-swarm \
+         /swarm manage -H tcp://$(boot2docker ip):9999 \
+         --tlsverify --tlscacert=/tls/ca.pem --tlscert=/tls/cert.pem --tlskey=/tls/key.pem \
+         token://$(cat ~/.docker/swarm-token)
+```
+
+Then set your local OS X Docker client settings to use the Docker Swarm port:
+
+```
+$ export DOCKER_HOST=tcp://$(boot2docker ip):9999
+## NOTE - this needs to be changed, but to do it, I need to generate some more certs
+$ unset DOCKER_TLS_VERIFY
+$ docker --tls info
+Containers: 31
+Nodes: 1
+ boot2docker: 192.168.59.103:2376
+$ docker --tls run hello-world
+```
+
+You can do this on multiple Boot2Docker hosts set up with TLS,
+by replacing the `/var/lib/boot2docker/tls/ca.pem` files in each one with one common
+ca certificate, and deleting the other files in the VM's `/var/lib/boot2docker/tls`
+directory.
+
+
```

Tell me what you think - I'll iterate next week on it a little more to replace the `--tls` with `DOCKER_TLS_VERIFY` again, and then add a second Docker to the TLS Swarm.

The text above was tested and worked on my 1.5.0rc2 b2d, using the larger debian based `svendowideit/docker-swarm` - using docker for the 2 long running tasks means we get automatic background, monitoring and logging, so :)

File: userguide.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+and expose them as a single "virtual" host.
+
+The Docker `swarm` manager can be interacted with using the Docker API, which means
+any tool which can communicate with a Docker Daemon using that API, can control
+a Docker swarm transparently: dokku, fig, krane, flynn, deis, docker-ui, shipyard,
+drone.io, Jenkins... and of course the Docker client itself.
+
+Like the other Docker projects, `swarm` follows the "batteries included but removable"
+principle. It ships with a simple scheduling backend out of the box. The goal is
+to provide a smooth out-of-box experience for simple use cases, and allow swapping
```
Comment (vieux) [inline userguide.md]: we should use the future here, as we don't support any other backend yet.

**Comment by fredlf** [comment]
LGTM

**Comment by SvenDowideit** [comment]
@vieux @aluzzardi please note that we're dependent on you guys merging docs changes - unless you want to give us merge power :)

**Comment by vieux** [comment]
LGTM

**Pull request merged**